src/main/java/org/websoso/WSSServer/dto/userNovel/UserNovelUpdateRequest.java

```diff
@@ -0,0 +1,30 @@
+package org.websoso.WSSServer.dto.userNovel;
+
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.Nulls;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import java.util.List;
+import org.hibernate.annotations.ColumnDefault;
+import org.websoso.WSSServer.domain.common.ReadStatus;
+import org.websoso.WSSServer.validation.UserNovelRatingConstraint;
+
+public record UserNovelUpdateRequest(
+        @UserNovelRatingConstraint
+        @ColumnDefault("0.0")
+        Float userNovelRating,
+        @NotNull(message = "읽기 상태는 null일 수 없습니다.")
+        ReadStatus status,
+        @Pattern(regexp = "\\d{4}-\\d{2}-\\d{2}", message = "시작 날짜는 yyyy-MM-dd 형식이어야 합니다.")
+        String startDate,
+        @Pattern(regexp = "\\d{4}-\\d{2}-\\d{2}", message = "종료 날짜는 yyyy-MM-dd 형식이어야 합니다.")
+        String endDate,
+        @JsonSetter(nulls = Nulls.AS_EMPTY)
+        @Size(max = 3, message = "매력 포인트는 최대 3개까지 가능합니다.")
+        List<String> attractivePoints,
+        @JsonSetter(nulls = Nulls.AS_EMPTY)
+        @Size(max = 10, message = "키워드는 최대 10개까지 가능합니다.")
+        List<Integer> keywordIds
```

Comment:
q3; 
필드를 기준으로 개행하는 것은 어떤가요!
```suggestion
        @UserNovelRatingConstraint
        @ColumnDefault("0.0")
        Float userNovelRating,
        
        @NotNull(message = "읽기 상태는 null일 수 없습니다.")
        ReadStatus status,
        
        @Pattern(regexp = "\\d{4}-\\d{2}-\\d{2}", message = "시작 날짜는 yyyy-MM-dd 형식이어야 합니다.")
        String startDate,
        
        @Pattern(regexp = "\\d{4}-\\d{2}-\\d{2}", message = "종료 날짜는 yyyy-MM-dd 형식이어야 합니다.")
        String endDate,
        
        @JsonSetter(nulls = Nulls.AS_EMPTY)
        @Size(max = 3, message = "매력 포인트는 최대 3개까지 가능합니다.")
        List<String> attractivePoints,
        
        @JsonSetter(nulls = Nulls.AS_EMPTY)
        @Size(max = 10, message = "키워드는 최대 10개까지 가능합니다.")
        List<Integer> keywordIds
```

Reply:
좋습니다!! 피드 등 다른 dto와 일관성이 안맞다고 생각해 개행을 추가하지 않았었는데 Notice에는 추가되어 있었네요. 나중에 dto 리팩토링 시에 통일시키면 좋을 것 같아요! 일단 요기 개행 추가해뒀습니다~